ruby/ql/test/library-tests/ast/calls/calls.rb

```diff
@@ -344,4 +344,15 @@ def foo(a, b, ...)
 foo(x: 42)
 foo(x:, novar:)
 foo(X: 42)
-foo(X:)
\ No newline at end of file
+foo(X:)
+
+# calls inside lambdas
+y = 1
+id = ->(x) { y }
```

Comment:
nit: I would call it e.g. `one`, `id` makes me expect that it is the identify function.

Reply:
Oh true, haha. I think I initially wrote the identity function and then changed it to be relevant to this fix...

Reply:
I'll fix this in a followup PR, so you don't have to re-approve this one.

Reply:
https://github.com/github/codeql/pull/8543